components/automate-ui/src/app/modules/infra-proxy/org-details/org-details.component.ts

```diff
@@ -76,59 +102,76 @@ export class CookbooksListComponent implements OnInit, OnDestroy {
       this.store.dispatch(new GetCookbooksForOrg({
         server_id: server_id, org_id: org_id
       }));
+      this.store.dispatch(new GetRoles({
+        server_id: server_id, org_id: org_id
+      }));
     });
 
     combineLatest([
       this.store.select(getStatus),
       this.store.select(updateStatus),
-      this.store.select(getAllCookbooksForOrgStatus)
+      this.store.select(getAllCookbooksForOrgStatus),
+      this.store.select(getAllRolesForOrgStatus)
     ]).pipe(
       takeUntil(this.isDestroyed)
-    ).subscribe(([getOrgSt, updateSt, getCookbooksSt]) => {
+    ).subscribe(([getOrgSt, updateSt, getCookbooksSt, getRolesSt]) => {
         this.isLoading =
-          !allLoaded([getOrgSt, getCookbooksSt]) || updateSt === EntityStatus.loading;
+          !allLoaded([getOrgSt, getCookbooksSt, getRolesSt]) || updateSt === EntityStatus.loading;
       });
 
     combineLatest([
       this.store.select(getStatus),
       this.store.select(orgFromRoute)
     ]).pipe(
-        filter(([getOrgSt, _orgState]) =>
-          getOrgSt === EntityStatus.loadingSuccess),
-        filter(([_getOrgSt, orgState]) =>
-          !isNil(orgState)),
-        takeUntil(this.isDestroyed)
-      ).subscribe(([_getOrgSt, orgState]) => {
-        this.org = { ...orgState };
-        this.updateOrgForm.controls['name'].setValue(this.org.name);
-        this.updateOrgForm.controls['admin_user'].setValue(this.org.admin_user);
-        this.updateOrgForm.controls['admin_key'].setValue(this.org.admin_key);
-      });
+      filter(([getOrgSt, _orgState]) =>
+        getOrgSt === EntityStatus.loadingSuccess),
+      filter(([_getOrgSt, orgState]) =>
+        !isNil(orgState)),
```

Comment:
Helps a bit to be concise both vertically and horizontally when possible.
```suggestion
      filter(([getOrgSt, _orgState]) => getOrgSt === EntityStatus.loadingSuccess),
      filter(([_getOrgSt, orgState]) => !isNil(orgState)),
```
but could also just combine them without loss of clarity:
```suggestion
      filter(([getOrgSt, orgState]) => getOrgSt === EntityStatus.loadingSuccess && !isNil(orgState)),
```
(Same comment applies a couple more times below in this file)